examples/pytorch/llm/llm_sft.py

```diff
@@ -62,37 +67,25 @@ def parse_args() -> Arguments:
     return args
 
 
-args = parse_args()
+COMMAND_LINE_MODE = True
```

Comment:
same as llm_infer

Reply:
Ok, thank you for your advice

Reply:
hello! The following error will be reported in the notebook, and I took a different approach to solve it. 

ValueError: Some specified arguments are not used by the HfArgumentParser: ['--ip=127.0.0.1', '--stdin=9008', 
'--control=9006', '--hb=9005', '--Session.signature_scheme="hmac-sha256"', 
'--Session.key=b"ee0d3388-b7c0-47e3-911b-169d49b3e834"', '--shell=9007', '--transport="tcp"', '--iopub=9009', 
'--f=/home/jintao/.local/share/jupyter/runtime/kernel-v2-109761Qud2IXHZmrdh.json']